VisionDemo/clients/TFDemo/controllers/client/onStart.js

```diff
@@ -0,0 +1,52 @@
+    canvas = $("#cameracanvas").get(0);    
+    ctx = canvas.getContext("2d");
+    
+    // This function creates an http object to execute the 
+    // tensorflow procedure. It uses the response to draw the 
+    // boxes and labels on the image that it was given.  
+    // It also disables and then renables 
+    // the client buttons while it is processing. 
+    function processImageGlobal(design, args) {
```

Comment:
Just call this function processImage and skip the `processImage = processImageGlobal` at the end of the file

Reply:
The line at the bottom is necessary so that the function has a global designation. If I remove it then it throws errors in other files as it is considered a local function.

Reply:
Does it work instead if you do:

```
var processImage = function(design, args) {
...
}
```

I thought if you used var to declare the function it would be globally visible, but maybe I'm wrong about that.

Reply:
I tried changing it to that but still got the same error, I know it's not pretty but I tried some other ways to get a cleaner global function and this is the only one that works.